packages/bpk-component-slider/package.json

```diff
@@ -20,7 +20,8 @@
     "react-slider": "^0.11.2"
   },
   "devDependencies": {
-    "bpk-component-rtl-toggle": "^1.1.88"
+    "bpk-component-rtl-toggle": "^1.1.88",
+    "create-react-class": "^15.6.3"
```

Comment:
I think this might not be quite right - looking at the docs for this module says it's intended for those using React without ES6, which isn't us: https://reactjs.org/docs/react-without-es6.html

What was the issue you saw before you added this dependency?

Reply:
Yeah this shouldn't be required at all.

Reply:
Will have a look with it removed and see why slider was failing. It was giving some errors to do with compilation but will double check

Reply:
So had a look at react-slider too itself and seems the issue exists there where it currently isn't supporting react 16: https://github.com/mpowaga/react-slider/issues/120 

`create-react-class` is listed as a peer dependency for `react-slider` so we need to add it to the project to run it, but can be a devDependency as it is only being thrown by storybook when running.

What do you think?

Reply:
Oooh - ok so we _should_ have this listed as a dependency then so that we handle it for consumers.

Going forward the usage of this third party module should be reconsidered.